CMakeLists.txt

```diff
@@ -131,6 +131,11 @@ install(
     PATTERN "openmp_wrappers" EXCLUDE)
 
 option(FILE_REORG_BACKWARD_COMPATIBILITY "Enable File Reorg with backward compatibility" ON)
+#get rid of any RPATH definations already
+set_target_properties(hipify-clang PROPERTIES INSTALL_RPATH "")
+#set RPATH for the binary
+set_target_properties(hipify-clang PROPERTIES LINK_FLAGS "-Wl,--disable-new-dtags -Wl,--rpath,${CPACK_PACKAGING_INSTALL_PREFIX}/lib" )
```

Comment:
1. This code is for linking by GNU linker only. Both options are neither supported by the Microsoft C/C++ link tool nor have analogues. Suggestion: hide the above code under `if(UNIX)` condition.
2. Setting `--rpath` to `${CPACK_PACKAGING_INSTALL_PREFIX}/lib` assumes the packaging. We need to cover the cases when hipify-clang is built (and installed/not installed by cmake), and not used from the package installed. We need to cover those cases because hipify-clang is actually an independent standalone open-source tool that might be used separately from ROCm HIP.
3. Btw, unit-testing with your PR is ok on Windows 10 (if (1) is fixed) and Ubuntu 21.10.